modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -34,8 +35,17 @@ def submit
           evss_data = evss_mapper_service(auto_claim).map_claim
           evss_service.submit(auto_claim, evss_data)
 
-          generate_526_pdf(pdf_data)
-
+          pdf_string = generate_526_pdf(pdf_data)
```

Comment:
I feel it would be better to have a private method to extract this logic from the controller.

Reply:
ultimately this will be moved to the sidekiq job. all of this logic will be extracted from the controller at that point.